src/dso_api/dynamic_api/routers.py

```diff
@@ -21,7 +21,39 @@
     from dso_api.lib.schematools.models import DynamicModel
 
 
-class DynamicRouter(routers.SimpleRouter):
+class DynamicAPIRootView(routers.APIRootView):
```

Comment:
Misschien het is beter om deze class naar de views te trekken.

Reply:
Had ik helemaal niet aan gedacht! :-) Na even twijfelen (want het hoort ook bij de router functionaliteit) heb ik het toch verplaatst.